cleaner/docker.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2022 Red Hat, Inc. and/or its affiliates.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cleaner
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"time"
+)
+
+// https://docs.docker.com/engine/api/latest/
+func (d Docker) GetClient() (*client.Client, error) {
+	return client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+}
+
+func (d Docker) GetClientRemoteFromEnv(host string) (*client.Client, error) {
+	return client.NewClientWithOpts(client.FromEnv, client.WithHost(host))
```

Comment:
in the readme you've stated that is mandatory to set DOCKER_HOST env and the other envs from the method above, not sure about these two methods here.

Reply:
It's mandatory if you want to connect to a remote registry, client.FromEnv use the info available to connect to the local or remote registry, it depensd on what it find, if nothing is find the defaults are all with local